src/core_systems/continuous_time_sde.jl

```diff
@@ -0,0 +1,245 @@
+using SciMLBase: SDEProblem, AbstractSDEIntegrator, __init, SDEFunction
+using StochasticDiffEq: SOSRA
+export CoupledSDEs
+
+###########################################################################################
+# DiffEq options
+###########################################################################################
+const DEFAULT_SDE_SOLVER = SOSRA() # default recommendation for additive noise
+const DEFAULT_STOCH_DIFFEQ = (alg=DEFAULT_SDE_SOLVER, DEFAULT_DIFFEQ_KWARGS...)
+
+# Function from user `@xlxs4`, see
+# https://github.com/JuliaDynamics/jl/pull/153
+function _decompose_into_sde_solver_and_remaining(diffeq)
+    if haskey(diffeq, :alg)
+        return (diffeq[:alg], _delete(diffeq, :alg))
+    else
+        return (DEFAULT_SDE_SOLVER, diffeq)
+    end
+end
+
+###########################################################################################
+# Type
+###########################################################################################
+
+# Define CoupledSDEs
+"""
+    CoupledSDEs <: ContinuousTimeDynamicalSystem
+    CoupledSDEs(f, g, u0 [, p, σ]; kwargs...)
+
+A stochastic continuous time dynamical system defined by a set of
+coupled ordinary differential equations as follows:
+```math
+d\\vec{u} = \\vec{f}(\\vec{u}, p, t) dt + \\vec{g}(\\vec{u}, p, t) dW_t
+```
+
+Optionally provide the overall noise strength `σ`, the parameter container `p` and initial time as keyword `t0`. If `σ` is provided, the diffusion function `g` is multiplied by `σ`.
```

Comment:
Notice that `σ` is not included anywhere in the equation above. Wouldn't it be simpler to explicitly add `σ` to the equation above and just say it defaults to 1?

Reply:
Yes I agree